2-Authorization/0-call-api-vanillajs/SPA/public/authPopup.js

```diff
@@ -2,6 +2,19 @@
 // configuration parameters are located at authConfig.js
 const myMSALObj = new msal.PublicClientApplication(msalConfig);
 
+myMSALObj.initialize().then(() => {
```

Comment:
Is this the correct copy/paste from the redirect changes to support popup?

Reply:
Yes it is, this is how it is also done in [1-Authorization/0-call-api-vanillajs](https://github.com/Azure-Samples/ms-identity-ciam-javascript-tutorial/blob/main/1-Authorization/0-sign-in-vanillajs/App/public/authPopup.js) authPopup.js.

Reply:
Pretty sure this isn't needed for popup flows.  @sameerag are the samples in the MSAL.js repo all correct?  If so we might need a pass at the azure samples as well.

Reply:
We need `initialize()` for any 2.x usage. Which MSAL version is it use here?

Reply:
On a closer look, if you want to avoid `initialize()` call, please use `createPublicClientApplication()` instead. Docs [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/v2-migration.md)

Reply:
The versions of msal-browser are 3.16.0 and 3.17.0 for the 1-authorization & 2-Authorization vanilla js apps respectively.

Reply:
@sameerag

Reply:
I would recommend using `createPublicClientApplication()` as a pattern, we are moving towards it and will deprecate `new PublicClientApplication()` potentially in 4.x.

And as for `handleRedirectPromise()` you will not need it if it is only popUp.